src/style.css

```diff
@@ -0,0 +1,93 @@
+html {
+  font-weight: 300;
+}
+
+body {
+  margin-top: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
first we import fonts - then apply them
move html selector below import and add fon-family to htlm selector

Reply:
done